src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/SettlementOfferTest.java

```diff
@@ -216,9 +216,7 @@ private Claim createClaimWithResponse(User defendant) {
         );
 
         commonOperations.linkDefendant(
```

Comment:
The same here, linking will never work here since created defendant has no 'letter-' role.

Is that test passing at all? It should be impossible in CCD world to submit defence without access to the case.